_docs/faq.md

```diff
@@ -0,0 +1,253 @@
+---
+docid: faq
+title: FAQ / Troubleshooting Help
+layout: docs
+permalink: /docs/faq.html
+---
+
+* [What operating systems are supported?](#what-operating-systems-are-supported)
+* [What languages does Caffe2 support?](#what-languages-does-caffe2-support)
+* [How do I use Caffe2 with my GPU?](#how-do-i-use-caffe2-with-my-gpu)
+* [What are all of these optional libraries used for?](#what-are-all-of-these-optional-libraries-used-for)
+* [Why do I get import errors in Python when I try to use Caffe2?](#why-do-i-get-import-errors-in-python-when-i-try-to-use-caffe2)
+* [Why isn't Caffe2 working as expected in Anaconda?](#why-is-caffe2-not-working-as-expected-in-anaconda)
+* [How do I fix error messages that are protobuf related?](#how-do-i-fix-error-messages-that-are-protobuf-related)
+* [How can I find a file, library, or package on my computer?](#how-can-i-find-a-file-library-or-package-on-my-computer)
+* [No module named caffe2_pybind11_state_gpu](#no-module-named-caffe2pybind11stategpu)
+* [My python kernel keeps crashing when using Jupyter](#my-python-kernel-keeps-crashing-when-using-jupyter)
+* [I still have a question, where can I get more help?](#i-still-have-a-question-where-can-i-get-more-help)
+
+## What operating systems are supported?
+
+Caffe2 is tested on
+
+* macOS Sierra or above
+* Ubuntu 14.04 and 16.04
+* CentOS 7
+* Windows 10
+* iOS and Android
+* Raspberry Pi
+* Nvidia Tegra
+
+## What languages does Caffe2 support?
+
+Caffe2 is written in c++ with a Python frontend. You can find all of the code on our [Github page](https://github.com/caffe2/caffe2).
+
+## How do I use Caffe2 with my GPU?
+
+Many of Caffe2's operators have CUDA implementations, allowing you to use Caffe2 with your Nvidia GPU. To install Caffe2 with GPU support, first install all the needed Nvidia libraries ([CUDA](https://developer.nvidia.com/cuda-downloads) and CuDNN) and then follow the [installation instructions](https://caffe2.ai/docs/getting-started.html).
+
+## What are all of these optional libraries used for?
+
+Caffe2 can has many optional dependencies, which add features to Caffe2's core functionality.
```

Comment:
Nit: Caffe2 can has many optional dependencies, which add features to -> Caffe2 has many optional dependencies, which extend